README.md

```diff
@@ -1,59 +1,76 @@
+      - [Prerequisites](#prerequisites)
+      - [Syllabus](#syllabus)
+      - [Next Step](#next-step)
+  * [Build Learning Activities](#build-learning-activities)
+  * [Evaluate the Learners Performance](#evaluate-the-learners-performance)
+  * [Provide Feedback on Performance.](#provide-feedback-on-performance)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
+
 # Building a Curriculum for VWC
 Keep in mind that laying out a well thought-out and comprehensive curriculum that achieves our organizational goals is an iterative process. We seek to continually improve the effectiveness of our courses by improving learning outcomes. 
 
 Primary focus at the beginning is on writing learning objectives and an overall content structure to support those objectives. 
 
-## 1. Overall Objective
-This describes the intended outcome of the course from the learners perspective. 
-
-*Ex. By completing this course the learner will have all the essential skills to be hired as a Frontend Developer.*
-
-TODO - Provide a guide for how to write a solid overall objective. 
+## Curriculum Structure
 
-## 2. Course Prerequisites
-What prior knowledge and skills should the learner have before taking this course. You may not be able to effectively determine this until you've completed the course design and development process. 
+### :factory: Folder Structure :factory:
+Repository is named after the course *ex: web-development-curriculum*
 
-## 3. Curriculum Structure
-Repository will be named after the course *ex web-development-curriculum*
-Folder for each top level module. 
-
-Includes a readme for that includes:
-- **Rationale** - Explain why learners should learn this and how it applies to them and the overall goal. 
-- **Objectives** - Inform the learner what they will be able to do when they complete the learning module. 
-- **Evaluation** - Explains how the module will be evaluated. 
-- **Resources** - Provides resources for completing activities. 
-- **Activities** - Gives the learners things to do. 
+Include a folder for each lesson within the module that will include that specific lessons activities and resources. 
 
-- **Next Step** - Describe what to do next. 
+Note: If you would like a specific folder order on Github displayed use a numbered prefix. As a rule of thumb start with a leading `0` example: `01`, `02`, `03` etc... this way folders will display top to bottom.
 
-Include a folder for each lesson within the module that will include that specific lessons activities and resources. 
+> :warning: TODO: Include a sample file structure for people to use as a template.
 
-TODO Include a sample file structure for people to use as a template.
+### Root level READMEs
+Includes a root level readme that includes:
 
+#### Rationale
+Explain why learners should learn this and how it applies to them and the overall goal of the course.
 
-## 4. How to Write an Effective Rationale
```

Comment:
Removed the `How to` prefix. I don't think we want to build step by step guides but more general guidance and tips to increase curriculum development velocity while maintaining quality.